thirdparty_containers/derby/playlist.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
+	<test>
+		<testCaseName>derby_test_junit_all</testCaseName>
+	<command>docker run --rm -v $(JDK_HOME):/java  adoptopenjdk-derby-test:latest ; \
+		$(TEST_STATUS)</command>
+		<subsets>
+			<subset>SE80</subset>
```

Comment:
If you drop this subset, then do you not get past the "needs Java9+" error msg?

Reply:
Hi @ShelleyLambert , yes we would, but I was under the impression that we want to test Java 8 and above with the external tests due to the fact that we have only two Java 8 builds for external tests at Adopt (openjdk8_hs* and openjdk8_j9*). Please correct me if I am wrong.

Reply:
As per our slack conversation just now, we can add openjdk9* and openjdk10* builds at any time, I just had not gotten to it yet :)  

So for now, you could pull material from HEAD for Derby, and run them against Java9 SDKs

Reply:
Yes, I am on it. Thanks for clarifying.

Reply:
Updated Derby test scripts to pull Derby from HEAD. Tested here: https://urv-jenkins.canlab.ibm.com:9095/view/SV_Tests/job/test.personal/135/console